datamodel/src/main/java/com/hartwig/serve/datamodel/serialization/util/ActionableFileUtil.java

```diff
@@ -189,6 +211,8 @@ public static String toLine(@NotNull ActionableEvent event) {
                 .add(event.level().toString())
                 .add(event.direction().toString())
                 .add(setToField(event.evidenceUrls()))
+                .add(formatDate(event.date()))
```

Comment:
Curious why you make a function `formatDate` here, but for the line below you don't (although it's the same "kind of" code). I would think the `formatDate` function is not needed either, and you could just inline the code?